jupyterlab_hub/src/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+/// <reference path="../typings/url-join/url-join.d.ts"/>
+
+// For a Thenable reference in jupyter-js-services
+type Thenable<T> = Promise<T>
```

Comment:
This can go away soon, due to recent changes in jupyter-js-services. We'll update the cookie cutter soon.